spec/ParseObject.spec.js

```diff
@@ -268,18 +268,13 @@ describe('Parse.Object testing', () => {
     });
   });
 
-  it_exclude_dbs(['postgres'])("can set null", function(done) {
+  fit("can set null", function(done) {
```

Comment:
I will never submit a PR with this issue again :smile: 
https://github.com/kulshekhar/git-hooks/blob/master/parse-server/pre-commit